site/pages/about.md

```diff
@@ -0,0 +1,41 @@
+<!--
+.. title: About the NCAR Experimental Development Team
+.. slug: about
+.. date: 2019-10-21 15:46:50 UTC-06:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+The NCAR Experimental Development (Xdev) Team is a small team of software engineers
+dedicated to the advancement and development of modern tools and techniques
+in scalable data analysis for geoscientists.  The Xdev Team was created as part
+of a collaboration with the [Pangeo](https://pangeo.io) community.
+
+Currently, the Xdev Team consists of:
+
+- **Kevin Paul (Engineering Lead)**: Kevin is a scientist and software engineer at NCAR with a
+  specialization in developing parallel Python tools for data post-processing, most recently developing
+  tools to post-process the CESM contributions for CMIP6.
+- **Matthew Long (Science Lead)**: Matt is a Scientist in the Climate & Global Dynamics laboratory
```

Comment:
Matt's the only one with his job title capitalized -- I'd suggest either Scientist -> scientist here or "software engineer" -> "Software Engineer" for the rest of us (of the two I'd prefer the former, but really my personal preference is for consistency)

Reply:
I don't think the job title needs to be capitalized, so I'll choose the former option.